### PR TITLE
feat #CCOP-110: add rating support to CustomerReviewModule

### PR DESCRIPTION
**Pull request by YanBrovkin**

### CODE REVIEW

File: CustomerReviewsModule.Web/Scripts/blades/review-detail.tpl.html [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+﻿<div></div>
+<div class="blade-content __medium-wide">
     <div class="blade-inner">
         <div class="inner-block">
             <form class="form" name="detailForm">
                 <div class="form-group clearfix">
                     <div class="column">
                         <div class="form-group">
-                            <label class="form-label">{{ 'customerReviewsModule.blades.review-detail.labels.productId' | translate }}</label>
+                            <label class="form-label">{{ 'customerReviewsModule.blades.review-detail.labels.authorNickname' | translate }}</label>
```
Comment (eMazeika) [inline CustomerReviewsModule.Web/Scripts/blades/review-detail.tpl.html]: productId is an important field, it should be shown

File: CustomerReviewsModule.Data/Services/CustomerReviewService.cs [hunk 2]
```diff
@@ -72,5 +72,15 @@ public void DeleteCustomerReviews(string[] ids)
...
             }
         }
+
+        public int GetAverageRating(string productId)
+        {
+            if (productId.IsNullOrEmpty())
+                throw new ArgumentNullException(nameof(productId));
+            using (var repository = _repositoryFactory())
+            {
+                return Convert.ToInt32(repository.CustomerReviews.Select(r => r.Rating).AsEnumerable().Average());
```
Comment (eMazeika) [inline CustomerReviewsModule.Data/Services/CustomerReviewService.cs]: should filter by productId

Comment (YanBrovkin) [reply CustomerReviewsModule.Data/Services/CustomerReviewService.cs]: fixed

File: CustomerReviewsModule.Web/Scripts/blades/review-detail.tpl.html (hunk 1 above)
Comment (YanBrovkin) [reply CustomerReviewsModule.Web/Scripts/blades/review-detail.tpl.html]: fixed

**Pull request merged**